### ISSUE TITLE
Please add information specific to Windows plaform

### ISSUE THREAD

**Issue by vvivanloc** [MicrosoftDocs/vsts-docs#7712]
Thanks for this page.

* Putting all information into a single page is nice but it is pretty hard to follow. Please split it.

* For windows, it could be useful to indicate that you could use *Git Bash console* to generate ssh keys and *clip* to copy to the clipboard as written in [GitHub pages](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh).

**Comment by WilliamAntonRohm** [comment]
@vvivanloc -- Thanks for your feedback; we appreciate it. To make sure we have all the information needed to triage your issue, please let us know which topic you’re referencing, or provide more context around your general task.

**Comment by vvivanloc** [comment]
Ah, my bad. I thought the documentation page was referenced in my bug report.

Here is the page which deserve splitting and details specific for Windows plateform.
https://docs.microsoft.com/en-us/azure/devops/repos/git/use-ssh-keys-to-authenticate?view=azure-devops&tabs=current-page

Thanks.

**Comment by vvivanloc** [comment]
doh! Bad button.

**Comment by WilliamAntonRohm** [comment]
Thanks for the suggestion! The product teams aren’t always watching this repository, so to make sure it gets in front of them for triage, please [submit your idea]( https://developercommunity.visualstudio.com/content/idea/post.html?space=21) here.
